common/djangoapps/third_party_auth/utils.py

```diff
@@ -0,0 +1,43 @@
+from random import randint
+
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+from django.contrib.auth.models import User
+
+
+class GeneratorUsername(object):
+
+    def __init__(self):
+        self.separator_character = '_'
+        self.in_lowercase = True
+        self.random = True
+
```

Comment:
For reviewers: Please ignore this in this commit, it was just for testing purpose and not deal with site_configuration and settings still.